### PR TITLE
[KOGITO-4611] - Generic and tradicional JDBC implementation addon

### PR DESCRIPTION
**Pull request by swati-kale**
Signed-off-by: Swati Kale <swkale@redhat.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

**Review by cristianonicolai — APPROVED** [review]

File: process-postgresql-persistence-quarkus/pom.xml [hunk 1]
```diff
@@ -81,4 +72,85 @@
...
+        </property>
+      </activation>
+      <dependencies>
+        <dependency>
+          <groupId>org.kie.kogito</groupId>
+          <artifactId>postgresql-persistence-addon</artifactId>
+        </dependency>
+        <dependency>
+          <groupId>io.vertx</groupId>
+          <artifactId>vertx-pg-client</artifactId>
```
Comment (MarianMacik) [inline process-postgresql-persistence-quarkus/pom.xml]: No `quarkus-reactive-pg-client`?

File: process-postgresql-persistence-springboot/src/main/resources/application.properties [hunk 2]
```diff
@@ -1,9 +1,3 @@
-server.address=0.0.0.0
 
-kogito.persistence.type=postgresql
 
-#Details https://www.postgresql.org/docs/9.6/static/libpq-connect.html#LIBPQ-CONNSTRING
-kogito.persistence.postgresql.connection.uri=postgresql://kogito-user:kogito-pass@localhost:5432/kogito
```
Comment (MarianMacik) [inline process-postgresql-persistence-springboot/src/main/resources/application.properties:3]: This file is left empty with 3 lines. I suggest to either keep it with a comment that it is intentionally left blank and eventually replaced or remove it completely.

Comment (swati-kale) [reply process-postgresql-persistence-springboot/src/main/resources/application.properties:3]: ok

File: process-postgresql-persistence-quarkus/pom.xml (hunk 1 above)
Comment (swati-kale) [reply process-postgresql-persistence-quarkus/pom.xml]: yes, done.

**Review by MarianMacik — APPROVED** [review]

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Pull request merged**